### DISCUSSION TITLE
GPS Speed for 12 Black

### DISCUSSION THREAD

**Discussion by jsiver** [gopro/labs#1003] [Q&A]
The QR code generator option for GPS speed trigger only shows up to 11Black.  Does this feature not work on the 12Black?

**Comment by dnewman-gpsw** [comment]
HERO12 has no GPS receiver, so that Labs feature can't work.

**Comment by jsiver** [comment]
Thank you. I now purchased an 11Black as i need gps.  I'm trying to create a QR code that starts recording video at 30pmh and keeps recording until the speed is below 30mph.  The "start at gps velocity" option allows "start speed" and "end speed".  Does the end speed have to be faster than the start speed, or will i get the desired result with start speed of 30 and end speed of 29?

**Reply by dnewman-gpsw** [reply]
Greater or equal will trigger start.  You don't need to set an end speed if it is the same.
